package.json

```diff
@@ -95,12 +98,13 @@
     "glob": "^7.2.3",
     "husky": "^8.0.3",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.6.3",
+    "jest": "^29.6.4",
     "node-sass": "^7.0.3",
     "npm-run-all": "^4.1.5",
     "postcss": "^8.4.21",
     "prop-types": "^15.8.1",
     "react-intl": "^5.25.1",
+    "react-test-renderer": "^17.0.2",
```

Comment:
Let's try to not introduce `react-test-renderer` and use an RTL alternative test to using `ShallowRenderer`.

Reply:
Got it! Thanks, I'm gonna do that :)

Reply:
Issue with TableView.test.js
if I replace the shallow test with asFragment - it fails due to useDispatch not being mocked. If I mock it, it fails with the following error:

 Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

    Check the render method of `AsyncRemediationButton`.

Google says that's an incorrect import issue. I tried the proposed solutions (add curly braces) and it's not the "fix"

Reply:
The mock in `setupTests.js` is not correct. it should be:

```js
 jest.mock(
    '@redhat-cloud-services/frontend-components/AsyncComponent',
    () => ({
        __esModule: true,
        default: (props) => <div {...props}>AsyncComponent</div>,
    })
);
```

Reply:
Great catch! Thanks <3